frontend/src/modules/machinery/components/Translation.js

```diff
@@ -46,7 +46,7 @@ export default class Translation extends React.Component<Props> {
     render() {
         const { locale, sourceString, translation, isReadOnlyEditor } = this.props;
 
-        const types = translation.sources.map(source => source.type);
+        const types = translation.sources.map(source => source);
```

Comment:
This is equivalent to `const types = translation.sources`. No need to use a `map` here.

Reply:
Yeah I also wondered about this bit. Okay, will change it!